test/test_pf_distributed_slack.py

```diff
@@ -0,0 +1,42 @@
+import os
+import numpy as np
+import pypsa
+
+def normed(s): return s/s.sum()
+
+def test_pf_distributed_slack():
+    csv_folder_name = os.path.join(os.path.dirname(__file__), "../examples/scigrid-de/scigrid-with-load-gen-trafos/")
+
+    network = pypsa.Network(csv_folder_name)
+
+    #There are some infeasibilities without line extensions
+    network.lines.s_max_pu = 0.7
+    network.lines.loc[["316","527","602"],"s_nom"] = 1715
+    network.storage_units.state_of_charge_initial = 0.
+
+    network.lopf(network.snapshots, solver_name='cbc', formulation='kirchhoff')
+
+    #For the PF, set the P to the optimised P
+    network.generators_t.p_set = network.generators_t.p_set.reindex(columns=network.generators.index)
```

Comment:
is there a reason for first reindexing and then overwriting? Seem's to be a leftover of a the scipy-example-code somehow